### PR TITLE
nit: diff minimisation between asset hub runtimes

### PR DESCRIPTION
**Pull request by gilescope**
no functional changes (made one thing const in westend runtime).

This is just making the diffs between westend, polkadot and kusama asset hubs a bit cleaner.

### CODE REVIEW

**Review by PatricioNapoli — APPROVED** [review]

File: parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs [hunk 1]
```diff
@@ -111,7 +111,7 @@ pub type ForeignAssetsConvertedConcreteId = assets_common::ForeignAssetsConverte
 	(
 		// Ignore `TrustBackedAssets` explicitly
 		StartsWith<TrustBackedAssetsPalletLocation>,
-		// Ignore assets that start explicitly with our `GlobalConsensus(NetworkId)`, means:
+		// Ignore asset which starts explicitly with our `GlobalConsensus(NetworkId)`, means:
```
Comment (joepetrowski) [inline parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs:114]: But "that" is correct here. (https://www.grammarly.com/blog/which-vs-that/)

File: parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs [hunk 2]
```diff
@@ -118,8 +118,8 @@ pub type ForeignAssetsConvertedConcreteId = assets_common::ForeignAssetsConverte
 		// Ignore `TrustBackedAssets` explicitly
 		StartsWith<TrustBackedAssetsPalletLocation>,
 		// Ignore asset which starts explicitly with our `GlobalConsensus(NetworkId)`, means:
```
Comment (joepetrowski) [inline parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs:120]:
```suggestion
		// Ignore assets that start explicitly with our `GlobalConsensus(NetworkId)`; means:
```

File: parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs [hunk 3]
```diff
@@ -86,7 +86,7 @@ pub type CurrencyTransactor = CurrencyAdapter<
 	(),
 >;
 
-/// `AssetId/Balance` converter for `TrustBackedAssets``
+/// `AssetId/Balance` converter for `TrustBackedAssets`
```
Comment (joepetrowski) [inline parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs:89]:
```suggestion
/// `AssetId`/`Balance` converter for `TrustBackedAssets`.
```

File: parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs (hunk 1 above)
Comment (gilescope) [reply parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs]: Nice, though I went to a grammar school they did not teach grammar - for some reason it wasn't cool in the '80s.

**Review by joepetrowski — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Review by rphmeier — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by gilescope** [comment]
bot merge

**Pull request merged**